helm_deploy/hmpps-person-on-probation-user-api/values.yaml

```diff
@@ -44,5 +44,11 @@ generic-service:
     groups:
       - internal
 
+custommetrics:
+  enabled: true
+  scrapeInterval: 15s
+  metricsPath: /prometheus
+  metricsPort: http
+
 generic-prometheus-alerts:
-  targetApplication: hmpps-person-on-probation-user-api
+  targetApplication: hmpps-resettlement-passport-api
```

Comment:
are you sure target application is hmpps-resettlement-passport-api it seems wrong to me

Reply:
We already got separate dashboard for resettlement-passport-api  and this is specific for PLT (Person-on-probation-user)